ecommerce_api_client/tests/test_auth.py

```diff
@@ -43,7 +62,18 @@ def test_headers(self):
     @httpretty.activate
     def test_tracking_context(self):
         """ Verify the tracking context is enclosed in the token payload, when specified. """
-        self.assert_expected_token_value({'foo': 'bar'})
+        self.assert_expected_token_value(tracking_context={'foo': 'bar'})
+
+    @httpretty.activate
+    def test_issuer(self):
+        """ Verify that the issuer is enclosed in the token payload, when specified. """
+        self.assert_expected_token_value(issuer='http://example.com/oauth')
+
+    @httpretty.activate
+    @patch.object(JwtAuth, 'datetime_now', Mock(return_value=CURRENT_TIME))
+    def test_expires_in(self):
+        """ Verify the expiration date is enclosed in the token payload, when specified. """
+        self.assert_expected_token_value(expires_in=30)
```

Comment:
i think this would be a stronger test if a value other than the function's default (30) was used.